### PR TITLE
Use instance properties instead of static properties for css classes

### PR DESCRIPTION
**Pull request by julien**
We'd like to be able to customize the css classes used by this component.

### CODE REVIEW

**Comment by eduardolundgren** [comment]
LGTM. Maybe it is a breaking change?

**Comment by julien** [comment]
Good thinking

I should probaly do more testing and get back to you.

Thanks

**Comment by marcoscv-work** [comment]
Hey guys,

Julien helped me because we are using toggle in electric and we were not able to manage the standard bootstrap classes.

We had to do this temporally change:
https://github.com/Robert-Frampton/generator-electric/blob/lexicon/generators/quartz/templates/src/components/Sidebar.js#L11

With this changes we are able to change the classnames and by default it looks like it is working in the same way.

Thanks guys!

**Comment by eduardolundgren** [comment]
Yes, I am fine in having those changes. 
@jbalsas Could you analyze the impact of the change since it's a major. Thanks.

File: src/Toggler.js [hunk 1]
```diff
@@ -140,27 +140,40 @@ Toggler.STATE = {
...
 
-/**
- * The CSS class added to the content when it's collapsed.
- */
-Toggler.CSS_COLLAPSED = 'toggler-collapsed';
+	/**
+	 * The CSS class added to the content when it's collapsed.
+	 */
+	cssCollapsed: {
+		validator: value => core.isString(value),
```
Comment (jbalsas) [inline src/Toggler.js]: This can just be `core.isString`

File: src/Toggler.js [hunk 2]
```diff
@@ -140,27 +140,40 @@ Toggler.STATE = {
...
 
-/**
- * The CSS class added to the header when the content is collapsed.
- */
-Toggler.CSS_HEADER_COLLAPSED = 'toggler-header-collapsed';
+	/**
+	 * The CSS class added to the content when it's expanded.
+	 */
+	cssExpanded: {
+		validator: value => core.isString(value),
```
Comment (jbalsas) [inline src/Toggler.js]: Same, just `core.isString`

File: src/Toggler.js [hunk 3]
```diff
@@ -140,27 +140,40 @@ Toggler.STATE = {
...
 
-/**
- * The CSS class added to the header when the content is expanded.
- */
-Toggler.CSS_HEADER_EXPANDED = 'toggler-header-expanded';
+	/**
+	 * The CSS class added to the header when the content is collapsed.
+	 */
+	cssHeaderCollapsed: {
+		validator: value => core.isString(value),
```
Comment (jbalsas) [inline src/Toggler.js]: Just `core.isString`

File: src/Toggler.js [hunk 4]
```diff
@@ -140,27 +140,40 @@ Toggler.STATE = {
...
+		value: 'toggler-header-collapsed'
+	},
+
+	/**
+	 * The CSS class added to the header when the content is expanded.
+	 */
+	cssHeaderExpanded: {
+		validator: value => core.isString(value),
```
Comment (jbalsas) [inline src/Toggler.js]: Just `core.isString`

**Comment by julien** [comment]
Closing this pull request which is replaced by #2

**Pull request closed without merging**